static/templates/bot_avatar_row.handlebars

```diff
@@ -8,6 +8,9 @@
                 <button type="submit" class="btn open_edit_bot_form" data-sidebar-form="edit-bot" title="{{t 'Edit bot' }}" data-email="{{email}}">
                     <i class="icon-vector-pencil blue"></i>
                 </button>
+                <a type="submit" download="{{zuliprc}}" class="btn copy_bot_zuliprc" title="{{t 'Copy .zuliprc' }}" data-email="{{email}}">
```

Comment:
We should update the name of `.zuliprc` to `zuliprc` in the help text, since that's what actually gets downloaded.